src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -406,6 +422,12 @@ class SignUpModalController {
     // Save errors to local variable to inject into the form
     // Since errors are cleared on each step change
     this.signUpErrors = error.xhr.responseJSON.errorCauses
+    // Wait for the Okta widget to create the error message box and set the sign up button text
+    // before augmenting the error message box and resetting the sign up button text
+    this.$timeout(() => {
+      this.updateSignUpButtonText()
```

Comment:
Why have you moved the `updateSignUpButtonText` function into the `afterError` function?

This needs to go in the `afterRender` function as the buttons on the steps is now "Sign up".

Reply:
You should leave the `updateSignUpButtonText` in the `afterRender` function and either call it again or create a new function that fixes the submit button, but only when it's on step 3.

Reply:
This is what I'm seeing on your branch when running it locally on an incognito window. The button isn't changed.

<img width="939" alt="Screenshot 2025-03-07 at 3 39 46 PM" src="https://github.com/user-attachments/assets/9c0a1658-dbb5-46c9-948a-df407a85c6f1" />

Reply:
Sorry, I don't fully understand your suggestion.

Due to the button not being changed, we need to ensure that`updateSignUpButtonText` runs on each page.

Reply:
I think what happened is I accidentally changed the `updateSignUpButtonText` call to `injectBackButton` and them removed it when you [pointed out](https://github.com/CruGlobal/give-web/pull/1163#discussion_r1985126680) I had two `injectBackButton` calls. I added back the `updateSignUpButtonText` call to `afterRender` since I didn't mean to remove it.